### PR TITLE
24368 - Update BEN Statement in NOA

### PR DESCRIPTION
**Pull request by meawong**
*Issue #:* /bcgov/entity#24368

*Description of changes:*
- Integrate enable-new-ben-statement feature flag 
- Update logic for displaying the BEN statement in NOAs
- Add default value for ff

Here is the NOA BEN Statement when _enable-new-ben-statement_ is **FALSE**:
image

and when _enable-new-ben-statement_ is **TRUE**:
image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: legal-api/src/legal_api/reports/report.py [hunk 1]
```diff
@@ -240,6 +240,8 @@ def _get_template_filename(self):
         return '{}.html'.format(file_name)
 
     def _get_template_data(self):
+        from legal_api.services import flags  # pylint: disable=import-outside-toplevel
```
Comment (vysakh-menon-aot) [inline legal-api/src/legal_api/reports/report.py:243]: is there any reason for importing it here?

Comment (meawong) [reply legal-api/src/legal_api/reports/report.py:243]: I followed how the import was done in the business.py file that I was referencing but I can make it a top-level import.

Comment (vysakh-menon-aot) [reply legal-api/src/legal_api/reports/report.py:243]: yes. please move to the top

File: legal-api/src/legal_api/reports/report.py [hunk 2]
```diff
@@ -38,14 +38,16 @@
...
+from legal_api.services import Flags, MinioService, VersionedBusinessDetailsService
 from legal_api.utils.auth import jwt
 from legal_api.utils.formatting import float_to_str
 from legal_api.utils.legislation_datetime import LegislationDatetime
 
 
 OUTPUT_DATE_FORMAT: Final = '%B %-d, %Y'
 
+flags = Flags()
+
```
Comment (vysakh-menon-aot) [inline legal-api/src/legal_api/reports/report.py:50]: we are not suppose to re initiate here

File: legal-api/src/legal_api/reports/report.py [hunk 3]
```diff
@@ -38,14 +38,16 @@
 )
 from legal_api.models.business import ASSOCIATION_TYPE_DESC
 from legal_api.reports.registrar_meta import RegistrarInfo
-from legal_api.services import MinioService, VersionedBusinessDetailsService
+from legal_api.services import Flags, MinioService, VersionedBusinessDetailsService
```
Comment (vysakh-menon-aot) [inline legal-api/src/legal_api/reports/report.py:41]: Did you try this? if you are getting error then move the import to _get_template_data function
```suggestion
from legal_api.services import flags, MinioService, VersionedBusinessDetailsService
```

Comment (meawong) [reply legal-api/src/legal_api/reports/report.py]: Yes this works, just had to move the import order due to this:

image

**Review by vysakh-menon-aot — APPROVED** [review]

**Comment by severinbeauvais** [comment]
Good job, Megan!

**Pull request merged**